airtable.gemspec

```diff
@@ -1,28 +1,29 @@
-# coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'airtable/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "airtable"
-  spec.version       = Airtable::VERSION
-  spec.authors       = ["Nathan Esquenazi", "Alexander Sorokin"]
-  spec.email         = ["nesquena@gmail.com", "syrnick@gmail.com"]
-  spec.summary       = %q{Easily connect to airtable data using ruby}
-  spec.description   = %q{Easily connect to airtable data using ruby with access to all of the airtable features.}
-  spec.homepage      = "https://github.com/nesquena/airtable-ruby"
-  spec.license       = "MIT"
+  spec.name        = 'airtable'
+  spec.version     = Airtable::VERSION
+  spec.authors     = ['Nathan Esquenazi', 'Alexander Sorokin']
+  spec.email       = ['nesquena@gmail.com', 'syrnick@gmail.com']
+  spec.summary     = 'Easily connect to airtable data using ruby'
+  spec.description = 'Easily connect to airtable data using ruby with access to all of the airtable features.'
+  spec.homepage    = 'https://github.com/nesquena/airtable-ruby'
+  spec.license     = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ["lib"]
+  spec.files                 = `git ls-files -z`.split("\x0")
+  spec.bindir                = 'exe'
+  spec.executables           = `git ls-files -- exe/*`.split("\n").map {|f| File.basename(f)}
```

Comment:
is `exe/` better than `bin/` ?

Reply:
Yes. More info there - http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html